src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -53,16 +57,36 @@ const isValidAddress = (protocolVersion: ProtocolVersion, addressStr: string) =>
   }
 };
 
-const isReservedAddress = (ip: string, protocolVersion: ProtocolVersion) => {
+const isReservedAddress = (ip: string, protocolVersion: ProtocolVersion, isDnsAddress = false) => {
   try {
+    console.log(protocolVersion);
+
     if (protocolVersion === ProtocolVersion.ipv4) {
-      return ip === RESERVED_LOCAL_HOST_IPS.ipv4 || ip === RESERVED_CATCH_ALL_IPS.ipv4;
+      if (isDnsAddress) {
+        return (
+          ip === RESERVED_DNS_LOCALHOST_IP ||
+          ip === RESERVED_DNS_BROADCAST_IP ||
+          ip === RESERVED_LOCAL_HOST_IPS.ipv4 ||
+          ip === RESERVED_CATCH_ALL_IPS.ipv4
+        );
+      } else {
+        return ip === RESERVED_LOCAL_HOST_IPS.ipv4 || ip === RESERVED_CATCH_ALL_IPS.ipv4;
+      }
     } else {
       const ipv6Address = new Address6(ip);
-      return (
-        compareIPV6Addresses(new Address6(RESERVED_LOCAL_HOST_IPS.ipv6), ipv6Address) ||
-        compareIPV6Addresses(new Address6(RESERVED_CATCH_ALL_IPS.ipv6), ipv6Address)
-      );
+      if (isDnsAddress) {
+        return (
+          ip === RESERVED_DNS_LOCALHOST_IP ||
```

Comment:
We need to use the ipv6 equivalent to ipv4 broadcast etc addresses to compare here.